platform/ui/src/components/ViewportOverlay/ViewportOverlay.tsx

```diff
@@ -3,17 +3,25 @@ import classnames from 'classnames';
 
 import './ViewportOverlay.css';
 
+// The overlay-top and overlay-bottom classes are explicitly needed to offset
+// the overlays (i.e. via absolute positioning) such the ViewportActionCorners
+// have space for its child components.
+// ToDo: offset the ViewportOverlay automatically via css to account for the
+// space needed for ViewportActionCorners.
 const classes = {
-  topLeft: 'top-viewport left-viewport',
-  topRight: 'top-viewport right-viewport-scrollbar',
-  bottomRight: 'bottom-viewport right-viewport-scrollbar',
-  bottomLeft: 'bottom-viewport left-viewport',
+  topLeft: 'overlay-top left-viewport',
+  topRight: 'overlay-top right-viewport-scrollbar',
+  bottomRight: 'overlay-bottom right-viewport-scrollbar',
+  bottomLeft: 'overlay-bottom left-viewport',
 };
 
 const ViewportOverlay = ({ topLeft, topRight, bottomRight, bottomLeft, color }) => {
   const overlay = 'absolute pointer-events-none viewport-overlay';
   return (
-    <div className={classnames(color ? color : 'text-primary-light')}>
+    <div
+      className={classnames(color ? color : 'text-aqua-pale', 'text-[13px]', 'leading-5')}
+      style={{ textShadow: '0.8px 0.8px 0.5px rgba(0, 0, 0, 0.75)' }}
```

Comment:
Can we refactor this into the Tailwind config so that we can use it elsewhere? I mean the shadow. You know how we do custom colors in there, and we can use the name elsewhere. I'm pretty sure there must be an option for custom `textShadow`.

Reply:
Unfortunately there is no default support for textShadow, so we would have to do something like this... https://www.hyperui.dev/blog/text-shadow-with-tailwindcss. Alternatively, we could simply create a class that references a css rule. Thoughts?